collections/_article/dev-snapshot-godot-3-6-beta-2.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Dev snapshot: Godot 3.6 beta 2"
+excerpt: "Another beta build for Godot 3.6, implementing important bug fixes and some new features for existing games in production."
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/dev-snapshot-godot-3-6-beta-2.jpg
+image_caption_title: Murtop
+image_caption_description: A game by hiulit
+date: 2023-05-25 12:00:00
+---
+
+While most of our development focus is on the [upcoming Godot 4.1](/article/release-management-4-1/), and maintenance releases for the stable 4.0 branch (with [4.0.3 published last week](/article/maintenance-release-godot-4-0-3/)), a handful of dedicated contributors still work as time permits on finalizing the 3.6 release, to bring important fixes and some welcome new features to users who prefer not to port their projects to Godot 4.
+
+We had a first [3.6 beta 1](/article/dev-snapshot-godot-3-6-beta-1/) build a month ago, and it's time for the second one. We've only merged around 50 changes since beta 1, but some of those are pretty substantial and have long been in development, so there's a lot to test!
+
+A lot of work is also being done to improve the Android editor port. We're looking for interested users to help test the beta snapshots via Google Play, and provide us with feedback and automated reports on potential issues. [You can join the testing group here to get access.](https://groups.google.com/g/godot-testers)
+
+[Jump to the **Downloads** section.](#downloads)
+
+You can also [try the Web editor](https://editor.godotengine.org/releases/3.6.beta2/).
+
+*The illustration picture for this article is from* [**Murtop**](https://games.hiulit.com/murtop/) by [hiulit](https://twitter.com/hiulit), *a fast-paced arcade game packed with action, as if it was taken out directly from the 80's, where Dig Dug meets Bomberman. It's available on [Nintendo Switch](https://www.nintendo.com/store/products/murtop-switch/), [Steam](https://store.steampowered.com/app/2148170/Murtop/), and [itch.io](https://hiulit.itch.io/murtop).*
+
+## What's new
+
+See the [curated changelog](https://github.com/godotengine/godot/blob/3.x/CHANGELOG.md) for a selection of some of the main changes since Godot 3.5.2. We now also have a great [interactive changelog](https://godotengine.github.io/godot-interactive-changelog/#3.6-beta2) you can use to review to review the changes since the previous beta, with convenient links to the relevant PRs on GitHub.
```

Comment:
```suggestion
See the [curated changelog](https://github.com/godotengine/godot/blob/3.x/CHANGELOG.md) for a selection of some of the main changes since Godot 3.5.2. We now also have a great [interactive changelog](https://godotengine.github.io/godot-interactive-changelog/#3.6-beta2) you can use to review the changes since the previous beta, with convenient links to the relevant PRs on GitHub.
```

Reply:
Fixed.